.github/workflows/deploy.yml

```
@@ -0,0 +1,81 @@
+# This workflow is used to deploy the app to a V-Server.
+# It is triggered by a push to the main branch or a pull request to the main branch.
+# It is also triggered by a push to the working branch.
+# It is also triggered by a pull request to the working branch.
+# It is also triggered by a push to the main branch.
+# It is also triggered by a pull request to the main branch.
+# It is also triggered by a push to the working branch.
+# It is also triggered by a pull request to the working branch.
+name: Deploy to V-Server
+
+on:
+  push:
+    branches: [ main, conduit-cicd-pipeline]
```

You don't need to have the branch trigger for the feature branch. As long as a PR is open the workflows will be triggered due to the next trigger.

```suggestion
    branches: [ main ]
```

---

You're right! Thanks for the suggested change.